publish_unit_test_results.py

```diff
@@ -735,7 +735,7 @@ def write_stats_file(stats, filename) -> None:
         f.write(json.dumps(stats))
 
 
-def main(token: str, event: dict, repo: str, commit: str, files_glob: str, check_name: str,
+def main(token: str, event: dict, repo: str, commit: str, files_glob: str, issue_comment_title: str, check_name: str,
```

Comment:
Can we call this `comment_title` throughout the file? The `title` inside `publish_comment` is fine. And can `check_name` come first, then `comment_title`?

Reply:
Sure, i just thought to name it differently since it shadows the name from the outer scope

Reply:
You are right. I am going to refactor that code soon, no shadowing then.